changelog_unreleased/typescript/13764.md

```diff
@@ -1,10 +1,8 @@
-#### Support TypeScript 4.9 (#13764, #13783 by @sosukesuzuki)
+#### Support TypeScript 4.9 (#13764, #13783, #13872 by @sosukesuzuki)
```

Comment:
How about this header instead?

```suggestion
#### Support TypeScript 4.9 `satisfies` operator (#13764, #13783, #13872 by @sosukesuzuki)
```

```suggestion
#### Support TypeScript 4.9 [`satisfies`](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/#satisfies) operator (#13764, #13783, #13872 by @sosukesuzuki)
```

Otherwise we start with an overpromise. Instead of adding a header for `Auto-Accessors in Classes` we can say something like:

```markdown
[Auto-Accessors in Classes](https://devblogs.microsoft.com/typescript/announcing-typescript-4-9/#auto-accessors-in-classes) will be supported in an upcoming 2.8 patch release. We have de-scoped them for now to release `satisfies` operator sooner.
```

Reply:
Isn't it already supported by `babel-ts` parser? User can switch to use it temporarily.

Reply:
`@babel/parser` can parse auto accessors. But it cannot be used with type annotations to we need to wait https://github.com/babel/babel/pull/15209